proposals/lip-0044.md

```diff
@@ -127,32 +124,69 @@ The validator account holds the generator and BLS keys of a registered validator
 * `generatorKey`: The public key whose corresponding private key is used to sign blocks generated by the validator.
 * `blsKey`: The validator BLS key is the public BLS key whose corresponding private key is used to sign certificates.
 
-#### Generator List Substore
+#### Validators Params Substore
 
 ##### Substore prefix, Store Key, and Store Value
 
-* The substore prefix is set to `SUBSTORE_PREFIX_GENERATOR_LIST`.
+* The substore prefix is set to `SUBSTORE_PREFIX_VALIDATORS_PARAMS`.
 * The store key is set to empty bytes.
-* The store value is set to the _generator list_ data structure, serialized using the JSON schema `generatorListSchema`, presented [below](#json-schema-1).
-* Notation: For the rest of this proposal let `generatorList` be the entry in the generator list substore (for notational convenience, we refer directly to the array of addresses, without explicitly writing `generatorList.addresses`).
+* The store value is the serialization of an object following the JSON schema `validatorParamsSchema` defined below.
+* Notation: For the rest of this proposal let `validatorsParams` be the entry in the validators params substore.
```

Comment:
sometimes it's "validatorsParams" and sometimes "validatorParams"

Reply:
It should now be "validatorParams" everywhere.